New Post: Add-plugins, Add-mods

theTWIXhunter opened the pull request.

iamdeka commented on 2024-07-10-install-mods.md:
```diff
@@ -0,0 +1,53 @@
+tags: Mods
+permalink: minecraft/modifications/mods/add-mods
+description: Learn how to upload and install Mods to your Minecraft Java server.
+author: The_twix_hunter
```
`theTWIXhunter` looks better

theTWIXhunter replied:
Oh, I will note (and change) that Will

iamdeka commented on _drafts/minecraft/plugins/2024-07-09-install-plugins.md:
```diff
@@ -0,0 +1,58 @@
+tags: Plugins
+permalink: minecraft/modifications/plugins/add-plugins
+description: Learn how to upload and install plugins to your Minecraft Java server.
+author: The_twix_hunter
```
`theTWIXhunter` looks better

iamdeka commented on 2024-07-10-install-mods.md:
```diff
@@ -0,0 +1,53 @@
+> - the modloader needed (any forge Mod will work on neoforge servers)
+> and make sure those are the same as your server
+
+4. Navigate to the [FileManager](https://client.falixnodes.net/server/filemanager?dir=/) by hovering over "Manage" and clicking ["FileManager"](https://client.falixnodes.net/server/filemanager?dir=/) or open your SFTP program like FileZilla.
```
Please, don't write File Manager like that. Hyperlinks can have spaces.

iamdeka commented:
You also put the articles in the wrong folders
They should be in `/content/_posts`

iamdeka commented:
The dates and file names for linking are also mismatching. Please pay more attention to where you're putting the articles, and how they are named

theTWIXhunter commented:
wait, I commited the old files again

theTWIXhunter commented:
whoeps wrong file

theTWIXhunter commented:
ok, recommited your changes

iamdeka reviewed (changes requested):
...

![dGz14o0Q0V](https://github.com/user-attachments/assets/b0ef8d8b-0e90-415a-a083-2da3f34f9c00)

theTWIXhunter commented:
eurgg

theTWIXhunter commented:
I don't have a mc/java in my main

theTWIXhunter commented:
fixed it (hope fullt)

iamdeka commented:
> fixed it (hope fullt)

I don't see any commits

theTWIXhunter commented:
> I don't see any commits

I committed it

theTWIXhunter commented:
Committed is a small word, I committed it 6 times (I moved them to Java/mods and Java/plugins and I renamed them to 2024-07-27 (I made them on 10 that’s why they where called that on my pc)

iamdeka commented:
![m0E9xVMdFX](https://github.com/user-attachments/assets/cf701dde-88a2-47df-8dc2-34c6ee524769)

Put them in /content/....

iamdeka reviewed (changes requested):
⬆️

theTWIXhunter commented:
looks like they are now in /content/

theTWIXhunter commented:
I didn't close this

iamdeka commented:
> deleted the branch FalixNodes-Software:main

🤦

KuroiJigoku commented:
did bro just- deleted the branch that had these stuffs 🤦

theTWIXhunter commented:
> did bro just- deleted the branch that had these stuffs 🤦

I messed up some articles between diffrt Branches so I just reuploaded to a new branch
